packages/astro/src/cli/index.ts

```diff
@@ -38,6 +38,7 @@ function printHelp() {
 	title('Flags');
 	table(
 		[
+			['--host [optional host address]', 'Expose server on network'],
```

Comment:
nit: this feels a little verbose for just a summary. What about just `'--host [address]'` or `'--host [ip]'`? imo advanced details can be left to the documentation.

Reply:
Fair! Just wanted to clarify that it's optional, so users know `--host` could be a boolean flag. Maybe `--host [optional IP]` is a good compromise?